app/server/locale/ussd.en.yml

```diff
@@ -1,30 +1,30 @@
 en:
   kenya:
-    initial_language_selection: >
+    initial_language_selection: |
```

Comment:
this preserves the newlines

Reply:
I was just wondering. When I used the africa is talking sandbox it really looked like it simulated a smartphone. Would it also work on a feature phone?

Reply:
yep, feature phone interface should be the same